src/plugin.js

```diff
@@ -1,6 +1,7 @@
 import { resolve, dirname, relative } from 'path';
 import { ResolverFactory, SyncNodeJsInputFileSystem } from 'enhanced-resolve';
 import { parse } from 'babylon';
+import template from 'lodash.template';
```

Comment:
IMHO it's better to use `import template from 'lodash/template'` as modern lodash already modular and I see no need in distinct libraries like 'lodash.template' now.